tests/beman/iterator_interface/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ target_sources(
   PRIVATE FILE_SET beman_iterator_interface_tests_headers TYPE HEADERS)
 
 target_link_libraries(
-  beman.iterator_interface.tests PRIVATE beman::iterator_interface GTest::gtest
+  beman.iterator_interface.tests PRIVATE beman::iterator_interface # XXX GTest::gtest
```

Comment:
The tests have a direct dependency on gtest, not just on gtest main.

Reply:
`GTest::main` links public to `Test::getest`

Reply:
Right, but if we changed the test main it would break the tests. This is the linking equivalent of Include What You Use;  relying on transitive links of direct dependencies means surprising build breaks in the future.

Reply:
If you want, no problem.